### DISCUSSION TITLE
AVIF support in the Trasformation Presets

### DISCUSSION THREAD

**Discussion by sandros94** [directus/directus#16750] [Archive]
So earlier I've noticed that sharp supports avif format, and I wanted to give it a try but noticed that it wasn't available in the format dropdown. Since it is possible to edit the raw value I forced avif and made a preset with it.

Now, it works flawlesly and the image quality loss is comparable to the one that I get with webp, so sharp has a good implementation of avif. But, I've also noticed the huge cpu resources needed to just convert a single file (as expected with av1 and avif in general). Is this the reason why avif is still not in the format dropdown?

**Comment by sandros94** [comment]
While testitng stuff I've noticed I cannot parse the lossless option 
![Additional Transformations Arguments](https://user-images.githubusercontent.com/13056429/206326332-2031d8a4-f65a-4ac5-9901-39a980c42e75.png)

and it gives me:    
![internal server error](https://user-images.githubusercontent.com/13056429/206326518-7115f7bf-4ac4-449b-b1f3-56b261fc548d.png)

**Comment by rijkvanzanten** [comment]
> Is this the reason why avif is still not in the format dropdown?

Not really! The main reason is that avif support didn't exist at the time that we implemented that format dropdown 🙂 

I'll create a new issue to get that added 👍🏻

**Reply by rijkvanzanten** [reply]
https://github.com/directus/directus/issues/17076 🙂

**Comment by sandros94** [comment]
Great, thanks!
